src/app/verification/[id]/page.tsx

```diff
@@ -48,7 +48,7 @@ export default function DocumentVerification({
 
   return (
     <>
-      <BookingInfoSidebar bookingInfo={data} boatInfo={boatInfo} />
+      <BookingInfoSidebar bookingInfo={data} boatInfo={boatInfo} showRemainingPayment={true} />
```

Comment:
I prefer another solution which is not a flag

This can be the param

   t("sidebar.booking_info", {
                  boat: boatInfo.Nombre,
                  date: moment(bookingInfo.Date).format("DD/MM/YY"),
                  payment: bookingInfo.OutstandingPayment || 0,
                  location: bookingInfo.BoatLocation,
                })
                
                OR
                
                  t("sidebar.booking_info_without_remain", {
                  boat: boatInfo.Nombre,
                  date: moment(bookingInfo.Date).format("DD/MM/YY"),
                  location: bookingInfo.BoatLocation,